Dockerfile

```diff
@@ -1,10 +1,18 @@
 FROM node:lts-alpine3.16
 # ref: https://hub.docker.com/_/node?tab=tags&name=lts-alpine
 
-LABEL maintainer="Jupyter Project <jupyter@googlegroups.com>"
```

Comment:
The maintainer label was deprecated I recall

Reply:
I thought it was the `MAINTAINER` directive that was deprecated, not the label.

Reply:
Yeah I recall that also the label was a deprecated practice. I've added the `org.opencontainers.image.authors` label now.